x-pack/plugins/integration_assistant/server/types.ts

```diff
@@ -103,7 +103,7 @@ export interface LogFormatDetectionState {
   header: boolean;
   ecsVersion: string;
   results: object;
-  additionalProcessors: ESProcessorItem[]; // # This will be generated in the sub-graphs
+  additionalProcessors: ESProcessorItem[]; // Generated in handleXXX nodes or subgraphs.
```

Comment:
`CSV` node is not in `handleXXX` format.. May be rename it?

Reply:
I've renamed it